exercises/en/test_05_12.py

```diff
@@ -0,0 +1,15 @@
+def test():
```

Comment:
In this question can you delete `axis = _____` from the `exc_05_12.py` file so it matches the test file? Thank you